languages/javascript/exercises/concept/arrays-basic/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@exercism/javascript-arrays-basic",
+  "description": "Exercism concept exercise on arrays-basic",
+  "author": "Ovidiu Miu",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/exercism/language-concepts.git",
+    "directory": "languages/javascript/exercises/concept/arrays-basic"
```

```suggestion
    "directory": "languages/javascript/exercises/concept/arrays"
```

Can you also rename the folder from `arrays-basic` to `array`?

Use `git mv` if you want it to _not_ show up as deletes and adds.